Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -47,7 +47,7 @@ public class NavigationViewportDataSource: ViewportDataSource {
     /**
      Value of default viewport padding.
      */
-    var viewportPadding: UIEdgeInsets = .zero
+    public var viewportPadding: UIEdgeInsets = .zero
```

Comment:
@latargere, this property is mostly used internally. `NavigationViewportDataSource` provides `NavigationViewportDataSource.options.followingCameraOptions.paddingUpdatesAllowed` flag that you can disable and then provide your own padding. Something like this:
```swift
if let navigationViewportDataSource = navigationMapView.navigationCamera.viewportDataSource as? NavigationViewportDataSource {
    navigationViewportDataSource.options.followingCameraOptions.paddingUpdatesAllowed = false
    navigationViewportDataSource.followingMobileCamera.padding = .zero
}
```

Would such approach work for you?